Sources/AdhanObjc.swift

```diff
@@ -127,30 +127,33 @@ import Foundation
 @objcMembers open class BACalculationParameters: NSObject {
     open var method: BACalculationMethod = .other
     open var fajrAngle: Double
+    open var maghribAngle: Double?
```

Comment:
let's add a test in the objc tests please. Also We're still missing a case in `BACalculationMethod`

Reply:
Done